rules/renaming/tests/Rector/Name/RenameClassRector/RenameClassRectorTest.php

```diff
@@ -41,6 +41,15 @@ public function testClassNameDuplication(): void
         $this->doTestFileInfo($fixtureFileInfo);
     }
 
+    /**
+     * @see https://github.com/rectorphp/rector/issues/5389
+     */
+    public function testDoNotImplementTwice(): void
+    {
+        $fixtureFileInfo = new SmartFileInfo(__DIR__ . '/FixtureDuplication/do_not_implement_twice.php.inc');
+        $this->doTestFileInfo($fixtureFileInfo);
+    }
```

Comment:
This should be part of normal tests in `/Fixtures`. No need to create standalone directory, unless PHP version si different.

The `do_not_implement_twice.php.inc` is telling the story well